### DISCUSSION TITLE
Gfxstream Vulkan virtualization: Can it be use in termux?

### DISCUSSION THREAD

**Discussion by Artewar67** [termux/termux-packages#21523] [Q&A] [answered]
After the news of merged of a new gfxstream based driver for Vulkan virtualization: https://www.phoronix.com/news/Mesa-Gfxstream-Merged
 I was wondering if this could work in termux.

I understand gfxstream is used for android since version 10 for google play games and cutterfish. I don't see any obvious problems supporting it for termux.

sorry for bad english

**Comment by TomJo2000** [comment, marked as answer]
We're not shipping Mesa 24.3 yet.
Our package is still at 24.0.8,
thus this feature is not currently available in Mesa on Termux.

It is also unlikely to be useful on Termux as it is intended for Desktops running for example an Android VM guest.
